ci-operator/step-registry/hypershift/mce/power/create/nodepool/hypershift-mce-power-create-nodepool-commands.sh

```diff
@@ -70,188 +66,45 @@ export IBMCLOUD_TRACE=true
 echo "$(date) Creating VSI in PowerVS instance"
 ibmcloud pi ins create ${POWERVS_VSI_NAME} --image ${POWERVS_IMAGE} --subnets ${POWERVS_NETWORK} --memory ${POWERVS_VSI_MEMORY} --processors ${POWERVS_VSI_PROCESSORS} --processor-type ${POWERVS_VSI_PROC_TYPE} --sys-type ${POWERVS_VSI_SYS_TYPE} --replicants ${HYPERSHIFT_NODE_COUNT} --replicant-scheme suffix --replicant-affinity-policy affinity
 
-# Applying mirror config
-echo "$(date) Applying mirror config"
-cat <<EOF | oc apply -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: mirror-config
-  namespace: multicluster-engine
-  labels:
-    app: assisted-service
-data:
-  registries.conf: |
-    unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
-
-    [[registry]]
-      location = "registry.stage.redhat.io"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io/rhacm2"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io/rhacm2"
-        insecure = false
-
-    [[registry]]
-      location = "registry-proxy.engineering.redhat.com"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io/multicluster-engine"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io/multicluster-engine"
-        insecure = false
-EOF
-
+MCE_VERSION=$(oc get "$(oc get multiclusterengines -oname)" -ojsonpath="{.status.currentVersion}" | cut -c 1-3)
+ARCH="ppc64le"
+if (( $(echo "$MCE_VERSION < 2.6" | bc -l) )); then
+ # Support for using ppc64le arch added after MCE 2.6
+ ARCH="amd64"
+fi
 
-# Creating AgentServiceConfig
-CLUSTER_VERSION=$(oc get clusterversion -o jsonpath={..desired.version} | cut -d '.' -f 1,2)
-OS_IMAGES=$(jq --arg CLUSTER_VERSION "${CLUSTER_VERSION}" '[.[] | select(.openshift_version == $CLUSTER_VERSION)]' "${SHARED_DIR}/default_os_images.json")
-echo "$(date) Creating AgentServiceConfig"
+echo "$(date) Creating Nodepool"
 cat <<EOF | oc create -f -
-apiVersion: agent-install.openshift.io/v1beta1
-kind: AgentServiceConfig
+apiVersion: hypershift.openshift.io/v1beta1
```

Comment:
nit: We can also complete it using `hcp create nodepool`, the current method also looks good for me.